payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/BsbConfigTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.compose.ui.text.AnnotatedString
+import com.google.common.truth.Truth
+import com.stripe.android.view.BecsDebitBanks
+import org.junit.Test
+
+class BsbConfigTest {
+    // TODO: remove this list, not sure if we will remove the manually added bank list in BsbSpec.
```

Comment:
What was the intent here?

Reply:
For now, we're passing the manually added bank list to `BsbConfig`, same here for the tests. If we're going to load it from json in the future, we might need to re-define the list here.